AdyenDropIn/Utilities/ComponentManager.swift

```diff
@@ -78,11 +78,14 @@ internal final class ComponentManager {
             storedSection = ComponentsSection(header: .init(title: localizedString(.paymentMethodsStoredMethods,
                                                                                    configuration.localizationParameters),
                                                             editingStyle: editingStyle,
-                                                            style: ListSectionHeaderStyle()),
+                                                            style: configuration.style.listComponent.sectionHeader),
                                               components: storedComponents,
                                               footer: nil)
         } else {
-            storedSection = ComponentsSection(components: storedComponents)
+            storedSection = ComponentsSection(header: .init(title: localizedString(.paymentMethodsStoredMethods,
+                                                                                   configuration.localizationParameters),
+                                                            style: configuration.style.listComponent.sectionHeader),
+                                              components: storedComponents)
```

Comment:
i think i made a mistake when i told to do this, since in this case there shouldn't be any title for section.
i think you need to revert this :-), sorry!

Reply:
Okay.